### ISSUE TITLE
trouble unpacking jffs2 little endian

### ISSUE THREAD

**Issue by suqdiq** [andreax79/python-cstruct#3]
```
# jefferson -f 0.jffs2
Traceback (most recent call last):
  File "/usr/local/bin/jefferson", line 496, in <module>
    main()
  File "/usr/local/bin/jefferson", line 469, in main
    fs_list += scan_fs(content, cstruct.LITTLE_ENDIAN, verbose=args.verbose)
  File "/usr/local/bin/jefferson", line 299, in scan_fs
    dirent = Jffs2_raw_dirent()
  File "/usr/local/lib/python2.7/dist-packages/cstruct/__init__.py", line 301, in __init__
    self.unpack(CHAR_ZERO * self.__size__)
TypeError: unpack() takes exactly 3 arguments (2 given)
#
```

makes me sad :(

**Comment by andreax79** [comment]
Doh!

Please check the new version 1.4... there was an issue on the default value, but error was different :(

**Comment by suqdiq** [comment]
yeah i noticed 1.4 and tried it :) different error now :D 
```
# jefferson -f dest_file.little
Traceback (most recent call last):
  File "/usr/local/bin/jefferson", line 498, in <module>
    main()
  File "/usr/local/bin/jefferson", line 470, in main
    fs_list = scan_fs(content, cstruct.BIG_ENDIAN, verbose=args.verbose)
  File "/usr/local/bin/jefferson", line 290, in scan_fs
    unknown_node = Jffs2_unknown_node()
  File "build/bdist.linux-x86_64/egg/cstruct/__init__.py", line 298, in __init__
  File "/usr/local/bin/jefferson", line 71, in unpack
    cstruct.CStruct.unpack(self, data[:self.size])
TypeError: 'NoneType' object has no attribute '__getitem__'
```

if you wanna try this in the same scenario as me, im basically using jefferson (https://github.com/sviehb/jefferson) to unpack this image: http://wikisend.com/download/519176/lol.img

thank you!

**Comment by andreax79** [comment]
Fixed in 1.5
